Sources/SwiftDocC/Semantics/Symbol/Symbol.swift

```diff
@@ -115,7 +115,7 @@ public final class Symbol: Semantic, Abstracted, Redirected, AutomaticTaskGroups
     internal(set) public var extendedModule: String?
     
     /// Optional cross-import module names of the symbol.
-    internal(set) public var bystanderModuleNames: [String]?
```

Comment:
Since this is public API, can we deprecated this instead, and make it return `crossImportOverlayModule.bystanderModules`?